sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -40,15 +47,25 @@ export function buildQueueOptions(queue: CreateQueueOptions): InternalQueueOptio
     DuplicateDetectionHistoryTimeWindow: queue.duplicateDetectionHistoryTimeWindow,
     MaxDeliveryCount: getStringOrUndefined(queue.maxDeliveryCount),
     EnableBatchedOperations: getStringOrUndefined(queue.enableBatchedOperations),
-    AuthorizationRules: getRawAuthorizationRules(queue.authorizationRules),
+    
+    // TODO: found while syncing the ordering. This property is in .net, but not in ours?
+    //IsAnonymousAccessible: false,
+
+    AuthorizationRules: getRawAuthorizationRules(queue.authorizationRules),    
     Status: getStringOrUndefined(queue.status),
+    ForwardTo: getStringOrUndefined(queue.forwardTo),
+    UserMetadata: getStringOrUndefined(queue.userMetadata),
+
+    // TODO: found while syncing the ordering. This property is in .net, but not in ours?
+    // SupportOrdering: true,
```

Comment:
See https://github.com/Azure/azure-sdk-for-js/issues/11859#issuecomment-709649740

Reply:
Removed.